pkg/occlient/occlient.go

```diff
@@ -256,8 +257,17 @@ func isServerUp(server string) bool {
 		return false
 	}
 
+	timeout := ocRequestTimeout
```

Comment:
The only instance of `ocRequestTimeout` is being used here.. Why not remove the `ocRequestTimeout` constant completely? Then you can rename `timeout` to `ocRequestTimeout` and use it here.

Reply:
done